File: unit-2/sprint-2/day-2-RecyclerView/assignments/Idendity Card/app/src/main/java/com/example/idenditycard/MainActivity.java
package com.example.idenditycard;
import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.GridLayoutManager;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;
import android.widget.LinearLayout;
import android.widget.Toast;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    private RecyclerView recyclerView;
    private ArrayList<Model> modelList;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        recyclerView = findViewById(R.id.rvRecyclerView);
        buildData();
        setRecyclerAdapter();
    }

    private void buildData() {
        modelList = new ArrayList<>();
        for (int i = 0; i < 100; i++) {
            if (i % 3 == 0)
                modelList.add(new Model(R.drawable.jeff_bezos, "Microsoft", 64,"Business"));
            else if (i % 3 == 1)
                modelList.add(new Model(R.drawable.bill_gates, "Amazon", 56,"Business"));
            else if (i % 3 == 2)
                modelList.add(new Model(R.drawable.prateek_sukla, "Masai School", 31,"Business"));
        }
    }

    private void setRecyclerAdapter() {
        LinearLayoutManager layoutManager = new LinearLayoutManager(this);
        recyclerView.setLayoutManager(layoutManager);
        ModelAdaptor modelAdaptor = new ModelAdaptor(modelList);
        recyclerView.setAdapter(modelAdaptor);
    }
}

File: unit-5/Paggination/drive-download-20211012T092640Z-001/app/src/main/java/com/masai/pagingnetworkand03/MainActivity.kt
package com.masai.pagingnetworkand03

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.lifecycle.ViewModelProvider
import androidx.recyclerview.widget.LinearLayoutManager
import com.masai.pagingnetworkand03.ui.CharacterAdapter
import com.masai.pagingnetworkand03.ui.CharacterViewModel
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    private lateinit var characterViewModel: CharacterViewModel
    private lateinit var characterAdapter: CharacterAdapter

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        characterViewModel = ViewModelProvider(this).get(CharacterViewModel::class.java)
        setAdapter()

    }

    private fun setAdapter() {
        characterAdapter = CharacterAdapter()
        val linearLayoutManager = LinearLayoutManager(this)
        recyclerview.apply {
            layoutManager = linearLayoutManager
//            this.adapter = characterAdapter
        }
    }
}